examples/java/src/main/java/org/apache/beam/examples/MinimalWordCount.java

```diff
@@ -104,9 +104,11 @@ public static void main(String[] args) {
         .apply(Count.perElement())
         // Apply a MapElements transform that formats our PCollection of word counts into a
         // printable string, suitable for writing to an output file.
-        .apply(MapElements
```

Comment:
It's a bit unfortunate that there's no way to preserve manually formatted line-breaks like this, particularly for examples. But I think it's a small price to pay.